server/app.py

```diff
@@ -0,0 +1,16 @@
+import uvicorn
+from fastapi import FastAPI, File, UploadFile
+from handlers import calendarHandler
+
+app = FastAPI()
+
+@app.get("/calendarJson")
+async def showCalendar():
+    return calendarHandler.getCalendarJson()
+
+@app.post('/sendFile')
```

Comment:
The endpoint url is not RESTful, it should represent objects (in plural).
Request method `POST` means "submit" or "send" here, so it's sufficient to simply have `files` as url.

Reply:
Thanks for the suggestion! Changes made!